satellite/metabase/rangedloop/service.go

```diff
@@ -52,20 +53,25 @@ func NewService(log *zap.Logger, config Config, provider RangeSplitter, observer
 }
 
 // observerState contains information to manage an observer during a loop iteration.
-// Improvement: track duration.
 type observerState struct {
 	observer       Observer
 	rangeObservers []*rangeObserverState
+	// err is the error that occurred during the observer's Start method.
+	// If err is set, the observer will be skipped during the loop iteration.
+	err error
 }
 
 type rangeObserverState struct {
 	rangeObserver Partial
 	duration      time.Duration
+	err           error
```

Comment:
should we add also similar command like for observerState errro? something like `If err is set, the range observer will be skipped during the loop iteration.`